FirebaseAuthUI/FUIAuthStrings.m

```diff
@@ -137,6 +137,9 @@
     }
   }
   NSBundle *frameworkBundle = [FUIAuthUtils bundleNamed:bundleName];
+  if (frameworkBundle == nil) {
+    frameworkBundle = [NSBundle mainBundle];
+  }
```

Comment:
This change is optional and can be omitted, but should have no negative effects.